digdag-plugin-utils/src/main/java/io/digdag/util/RetryControl.java

```diff
@@ -10,21 +12,80 @@ public static RetryControl prepare(Config config, Config stateParams, boolean en
         return new RetryControl(config, stateParams, enableByDefault);
     }
 
+    private enum RetryIntervalType {
+        CONSTATNT("constant"), EXPONENTIAL("exponential");
+        String type;
+        private RetryIntervalType(final String type) {
+            this.type = type;
+        }
+        public static RetryIntervalType find(String value) {
+            for (RetryIntervalType r : RetryIntervalType.values()) {
+                if (r.type.equalsIgnoreCase(value)) {
+                    return r;
+                }
+            }
+            throw new ConfigException("Invalid retry_interval_type");
```

Comment:
I think that we'd better to show actual `value` value as the message like this. what do you think?
```
"Invalid retry_interval_type "  + value
```

Reply:
@muga Thank you for your pointing out. I fixed it.